Sources/TuistGenerator/Graph/Graph.swift

```diff
@@ -20,7 +20,7 @@ enum GraphError: FatalError {
     }
 }
 
-enum DependencyReference: Equatable, Hashable {
+enum DependencyReference: Equatable, Comparable, Hashable {
```

Comment:
👏

Reply:
Many many thanks @pepibumur for taking time to review it! 🙏